# Biostatistical Analysis of Omics Data in the R Programming Language

A common task when working with transcriptomic data is the identification of differentially expressed (DE) genes or tags between groups. In this workshop participants will learn how to perform biostatistical analysis in the R programming language, including pairwise and analysis of variance (ANOVA) like comparisons to identify significantly DE genes. (The analysis described in this workshop is downstream of the instructor's tutorial on Bioinformatics Analysis of Omics Data with the Shell & R, in which she provides a walkthrough of a simple bioinformatics workflow for quantifying transcriptomic data.)

## AFTER COMPLETING THIS WORKSHOP, PARTICIPANTS WILL
- Be able to replicate the statistical analysis of a published biological data set.
- Become comfortable working with quantified transcriptomic data.
- Learn what statistical comparisons are possible with the design of the experiment.
- Be able to perform pairwise comparisons with edgeR.
- Be able to perform ANOVA like comparisons with edgeR.
- Discover different types of plots for exploring expression data and analysis results.

## PREREQUISITES
- This workshop is designed for participants who are unfamiliar with statistical analysis.
- Participants are expected to be comfortable working in the R programming language. 

## PRIMARY TOOLS USED DURING THE WORKSHOP
- R software environment
- RStudio desktop software

## RESOURCES
- [Workshop Guide](https://morphoscape.wordpress.com/2022/08/09/downstream-bioinformatics-analysis-of-omics-data-with-edger/)
- [The Publication & Data](https://www.sciencedirect.com/science/article/pii/S0147651319302684)
- [Gene Transcription at Real-Time Speed](https://twitter.com/Innov_Medicine/status/1588155101075603456)
- [Exact or t-Tests Tutorial](https://online.stat.psu.edu/stat555/node/36/)
- [ANOVA Tutorial](https://online.stat.psu.edu/stat500/lesson/10)
